dali/test/python/test_RNNT_data_pipeline.py

```diff
@@ -0,0 +1,169 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import nvidia.dali
+import nvidia.dali.ops as ops
+import nvidia.dali.types as types
+import numpy as np
+import os
+import test_utils
+
+dali_extra_path = test_utils.get_dali_extra_path()
+
+
+# TODO remove
```

Comment:
?

Reply:
This PR is blocked by Pad operator. This is indicated in the description (TODO point). Once Pad is updated, the `_op_splicing` function and corresponding `PythonFunction` shall be removed. I've put it there to remember about it.

Reply:
You can add a reason for this ToDo then. I doubt that anyone will track it that to this PR description.

Reply:
Done